plugins/database/quota/src/org/apache/cloudstack/api/command/QuotaStatementCmd.java

```diff
@@ -119,10 +118,6 @@ public String getCommandName() {
 
     @Override
     public long getEntityOwnerId() {
-        Long accountId = _accountService.getActiveAccountByName(accountName, domainId).getAccountId();
```

Comment:
This can be kept to know who made the API call

Reply:
Ah, so that is what this is for.
I will create a log then.